priv/perf/config/dev.exs

```diff
@@ -5,10 +5,11 @@ config :ethereumex,
 
 config :load_test,
   child_chain_url: System.get_env("CHILD_CHAIN_URL"),
+  watcher_security_url: System.get_env("WATCHER_SECURITY_URL"),
```

Comment:
if we are default using env var, then probably put all this urls in the `config.exs` instead
So `dev.exs`, `stress.exs` does not need to re-declare.

For the `test.exs` I think we can hardcode to the localhost one, but that is my opinion